spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraRDD.scala

```diff
@@ -276,7 +276,37 @@ class CassandraRDD[R] private[connector] (
   lazy val tableDef = {
     Schema.fromCassandra(connector, Some(keyspaceName), Some(tableName)).tables.headOption match {
       case Some(t) => t
-      case None => throw new IOException(s"Table not found: $keyspaceName.$tableName")
+      case None => {
+        val clusterMetadata = connector.withClusterDo(_.getMetadata)
+        val keyspaces = clusterMetadata.getKeyspaces
+        val ksMap = keyspaces.groupBy(_.getName.toLowerCase).toMap
+        ksMap.get(keyspaceName.toLowerCase) match {
+          case Some(keyspaceList) => {
+            val possibleTables =
```

Comment:
Maybe we can include comparison with `StringUtils.getLevensteinDistance` or `StringUtils.getJaroWinklerDistance`? We would get the same nice behaviour as git has when you misspell some command.